src/benchmark.cc

```diff
@@ -1025,10 +1040,17 @@ std::unique_ptr<BenchmarkReporter> GetDefaultReporter() {
 } // end namespace internal
 
 size_t RunSpecifiedBenchmarks() {
```

Comment:
Good question: It's to avoid the "zero as null pointer constant" warning GCC emits in C++98 mode. 
Optimally we would use "= nullptr" as the default argument, but that's C++11 only, and using either "NULL" or "0" as the default value will trigger the warning.

I would prefer to use default arguments though.